tests/unittests/test_handler/test_schema.py

```diff
@@ -447,4 +448,27 @@ def test_all_integration_test_cloud_config_schema(self):
         if errors:
             raise AssertionError(', '.join(errors))
 
+
+class TestSchemaDocExamples:
+    @skipUnlessJsonSchema()
+    def test_schema_doc_examples(self):
+        ignored_files = [
+            'cloud-config-archive-launch-index.txt',
+            'cloud-config-archive.txt',
+        ]
+
+        examples_dir = Path(
+            __file__).parent.parent.parent.parent / 'doc' / 'examples'
+        # Make sure our path shenanigans found an actual directory
+        assert examples_dir.is_dir()
+
+        all_text_files = [f for f in examples_dir.glob('cloud-config*.txt')
+                          if f.name not in ignored_files]
```

Comment:
`cloud-config-archive` is a prefix for these files we want to exclude, in the same way that `cloud-config` is a prefix for the files that we do want to include. Can we drop the static list above if we make this something along the lines of `not f.name.startswith('cloud-config-archive')` instead?